### ISSUE TITLE
Feature request: changing the font of the whole table in ggtexttable

### ISSUE THREAD

**Issue by imaa9** [kassambara/ggpubr#578]
I would like to be able to use one of the existing tthemes ("light", in my case) but to change the font of the whole table to another font family. Is there already a straightforward way to do this or would a new feature need to be added? 
Thanks kindly for a fabulous set of packages, Inbar

**Comment by kassambara** [comment]
Yes — keep any built-in `ttheme()` and set the font family on the cell styles with the `fontfamily` argument (it's forwarded through `...` to the underlying text). For the whole table, set it on the column-name, row-name and body styles:

``` r
library(ggpubr)
df <- head(iris[, 1:4], 5)

my_theme <- ttheme(
  "light",
  colnames.style = colnames_style(fontfamily = "mono", fill = "grey90"),
  tbody.style    = tbody_style(fontfamily = "mono", fill = "white")
)
ggtexttable(df, rows = NULL, theme = my_theme)
```

![reprex](https://i.imgur.com/J9FVNYt.png)

Replace `"mono"` with any font family available on your system (e.g. `"Times"`, or a family registered via **showtext**/**systemfonts**); add `rownames.style = rownames_style(fontfamily = ...)` too if you keep row names. This preserves the "light" theme while changing the font throughout.

Closing as answered — reopen if you'd prefer a single top-level `font.family` argument.
